.github/workflows/static_analysis.yml

```diff
@@ -40,6 +40,7 @@ jobs:
             libxcb-xfixes0-dev \
             libxcb-xinerama0-dev \
             libxcb-xkb-dev \
+            git \
```

Comment:
Please could we keep this in alphabetical order. Also the commit should probably be `[ci]` instead of `[github]`.

Reply:
oops my fault :D haha

Reply:
Done.